### PR TITLE
deeplake v4 support

### PR DESCRIPTION
**Pull request by azat-manukyan**
- [ ] **PR title**: "community: Added support for Deeplake v4"

### CODE REVIEW

File: libs/community/langchain_community/vectorstores/deeplake.py
Comment (efriis) [inline libs/community/langchain_community/vectorstores/deeplake.py:1]: let's revert these changes. you're welcome to add a warning if the version doesn't start with 3 on init if this doesn't work with v4

**Review by efriis — COMMENTED** [review]
needs support for a larger review. if you're interested in workign with deeplake to publish a dedicated integration package - feel free to follow this guide as well!

https://python.langchain.com/docs/contributing/how_to/integrations/

**Comment by ccurme** [comment]
Closing as author is unresponsive but happy to re-open if we want to keep iterating on this.
